CHANGES

```diff
@@ -9,6 +9,9 @@ Highlights
   CL_DEVICE_TYPE_CUSTOM hardware accelerators which implement a memory
   mapped control interface.
 
+- Improved SPIR and SPIR-V support. clCreateProgramWithIL() implemented,
```

Comment:
Did SPIR-V work previously? Better advertise it more if this is the first release where it's usable.

Reply:
It did, sort of. It wasn't very usable (like SPIR) because of mangling.

Reply:
Ok, let's make it explicit.